src/pepr/operator/controllers/exemptions/exemptions.ts

```diff
@@ -1,9 +1,14 @@
 import { Log } from "pepr";
-import { policies } from "../../../policies/index";
-import { ExemptionElement, Matcher, Policy, UDSExemption } from "../../crd";
-
-type StoredMatcher = Matcher & { owner: string };
-type PolicyMap = Map<Policy, StoredMatcher[]>;
+import { PolicyMap, StoredMatcher } from "../../../policies";
+import { ExemptionElement, Policy, UDSExemption } from "../../crd";
+
+export enum WatchPhase {
```

Comment:
Was trying to use the enum from the fluent client but got errors when importing the type. so i just remade it here.